rotkehlchen/tests/db/test_reports.py

```diff
@@ -1,10 +1,76 @@
-from rotkehlchen.accounting.pnl import PnlTotals
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL, PnlTotals
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.constants import ONE, ZERO
+from rotkehlchen.constants.assets import A_DAI, A_ETH
+from rotkehlchen.db.filtering import ReportDataFilterQuery
 from rotkehlchen.db.reports import DBAccountingReports
 from rotkehlchen.db.settings import DBSettings
+from rotkehlchen.fval import FVal
 from rotkehlchen.tests.utils.constants import A_GBP
+from rotkehlchen.types import Location, Price, Timestamp
+from rotkehlchen.utils.misc import create_order_by_rules_list, timestamp_to_date
 
+TIMESTAMP_1_SECS = Timestamp(1741634066)
+TIMESTAMP_2_SECS = Timestamp(1741634100)
+ETH_PRICE_TS_1 = FVal('2000')
+ETH_PRICE_TS_2 = FVal('2200')
+
+EVENTS = [
```

Comment:
```suggestion
EVENTS: Final = [
```
Use the `Final` type annotation for global constants like this. Same for the timestamps and prices defined above.

Actually though, if you combine the tests into a single test, this should probably get defined within the test itself. Its always good to avoid global variables when possible.